tests/storage/datavolume.go

```diff
@@ -1272,7 +1272,7 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 			Expect(*vm.Spec.DataVolumeTemplates[0].Spec.Storage.StorageClassName).To(Equal(virtualMachinePreference.Spec.Volumes.PreferredStorageClassName))
 		})
 
-		It("should always use VM defined storage class over PreferredStorageClassName", func() {
+		It("should always use VM defined storage class over PreferredStorageClassName", decorators.Conformance, func() {
```

Comment:
Just like https://github.com/kubevirt/kubevirt/pull/13586/files#diff-aa884692b6a85018e3059a6e7623fe689a1a06f9d003cdf7fd24ba4bd76fa923R1265 , it sounds more of a unit test.

Reply:
Yep

Reply:
https://github.com/kubevirt/kubevirt/issues/13597